.ctags

```diff
@@ -0,0 +1,21 @@
+--langdef=GAP
+--langmap=GAP:.g.gd.gi.gap
+--regex-GAP=/([a-zA-Z0-9_]+) *:= *function/\1/f,function/
+--regex-GAP=/BindGlobal *\( *"([a-zA-Z0-9_]+)"/\1/v,value/
+--regex-GAP=/BIND_GLOBAL *\( *"([a-zA-Z0-9_]+)"/\1/v,value/
+--regex-GAP=/InstallMethod *\( *([a-zA-Z0-9_]+) *,/\1/m,method/
+--regex-GAP=/InstallOtherMethod *\( *([a-zA-Z0-9_]+) *,/\1/m,method/
+--regex-GAP=/InstallGlobalFunction *\( *([a-zA-Z0-9_]+) *,/\1/g,gfunction/
+--regex-GAP=/InstallValue *\( *([a-zA-Z0-9_]+) *,/\1/v,value/
+--regex-GAP=/InstallTrueMethod *\( *([a-zA-Z0-9_]+) *,/\1/t,truemethod/
+--regex-GAP=/<#GAPDoc *Label *= *"([a-zA-Z0-9_]+) *"/\1/d,documentation/
+--regex-GAP=/#Include *Label *= *"([a-zA-Z0-9_]+)"/\1/d,documentation/
+--regex-GAP=/DeclareCategory *\( *"([a-zA-Z0-9_]+)"/\1/c,category/
+--regex-GAP=/DeclareRepresentation *\( *"([a-zA-Z0-9_]+)"/\1/r,representation/
+--regex-GAP=/DeclareInfoClass *\( *"([a-zA-Z0-9_]+)"/\1/i,infoclass/
+--regex-GAP=/DeclareAttribute *\( *"([a-zA-Z0-9_]+)"/\1/a,attribute/
+--regex-GAP=/DeclareProperty *\( *"([a-zA-Z0-9_]+)"/\1/p,property/
+--regex-GAP=/DeclareSynonym *\( *"([a-zA-Z0-9_]+)"/\1/s,synonym/
+--regex-GAP=/DeclareSynonymAttr *\( *"([a-zA-Z0-9_]+)"/\1/s,synonym/
+--regex-GAP=/DeclareGlobalFunction *\( *"([a-zA-Z0-9_]+)"/\1/g,gfunction/
+--regex-GAP=/DeclareOperation *\( *"([a-zA-Z0-9_]+)"/\1/o,operation/
```

Comment:
I don't mind adding a ctags file, but let's do it in a separate PR, or at least in a dedicated commit inside this PR?

Reply:
Oh, I added that on accident. I have no idea how that got there. Weird. I've removed the file from the last commit.